### PR TITLE
Refactor MultiValuedTreeMap toString method

### PR DESCRIPTION
**Pull request by memojja**

### CODE REVIEW

File: extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/model/MultiValuedTreeMap.java [hunk 1]
```diff
@@ -181,21 +181,19 @@ public boolean equalsIgnoreValueOrder(MultiValuedTreeMap<Key, Value> vmap) {
...
         String delim = ",";
         for (Object name : keySet()) {
             for (Object value : get(name)) {
                 result.append(delim);
-                if (name == null) {
-                    result.append("null"); //$NON-NLS-1$
-                } else {
-                    result.append(name.toString());
-                }
+                result.append(name != null ? name : "null");
```
Comment (gastaldi) [inline extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/model/MultiValuedTreeMap.java:189]: Isn't easier to just do `result.append(name)`?

Comment (gastaldi) [reply extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/model/MultiValuedTreeMap.java:189]: Actually, better do it like: https://github.com/quarkusio/quarkus/blob/4278eb85b6e202840679dbf2e7c6400301b4f21f/independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/util/MultivaluedTreeMap.java#L111

Comment (memojja) [reply extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/model/MultiValuedTreeMap.java:189]: Thanks for your review, could you please review it again? I refactored except one subject. If i do `result.append(name)` , StringBuilder throwing `NullPointerException` when  doing `str.length()`

**Comment by geoand** [comment]
Thanks for this contribution!

When you are done, please squash the commits

**Comment by memojja** [comment]
it is done @geoand thanks

**Comment by gastaldi** [comment]
I'm not sure what changed here, shouldn't the brackets be included?

**Comment by gastaldi** [comment]
Closing as this PR doesn't seem to change anything

**Pull request closed without merging**